docs/docs/repo/labels.md

```diff
@@ -0,0 +1,49 @@
+## Labels
+
+Below is the list of labels and their descriptions used in Gossamer repository. 
+
+- **Epic** - Issue used to track development status of a complex feature, aggregates several issues. **Colour: #3E4B9E**
+- **Feature-branch** - pull request from feature branch to origin. **Colour: #8ceac8**
+- **Release** - pull request with next release changes. **Colour: #5319E7**
+- **`A-`**  Action needed label. Marks that there is a specific action needed for issue/PR  **Colours: #FBCA04**
+  - **A-tooBig** - issue or PR needs to be broken down to smaller parts.
+  - **A-stale** - issue or PR is deprecated and needs to be closed. 
+  - **A-blocked** - issue or PR is blocked  until something else changes.
+  - **A-triage** - issue desrciption needs refactor and/or labeled. 
+  - **A-debug** - issue requires detective debug work to figure out what's going wrong.
+  - **A-design** - issue requires design work to think about how it would best be accomplished.
+- **`T-`** Describes the type of issue or pull request. **Colour: #FEF2C0**
+  - **T-bug** - this issue covers unexpected and/or wrong behaviour. 
+  - **T-feat** - this issue/pr is a new feature or functionality.
+  - **T-enhancement** - this issue/pr covers improvement of existing functionality. 
```

Comment:
feature and enhancement is kind of the same

Reply:
I understand a feature as something new but an enhancement as an "improvement" of something that is already solved in our code. @P1sar was that your intention?

Reply:
Yeah, my meaning exactly as what Diego means